bazel/grpc_build_system.bzl

```diff
@@ -31,6 +31,10 @@ load("@build_bazel_rules_apple//apple:ios.bzl", "ios_unit_test")
 # The set of pollers to test against if a test exercises polling
 POLLERS = ["epollex", "epoll1", "poll"]
 
+# set exec_properties = LARGE_MACHINE, to run the test on a large machine
+# see //third_party/toolchains/machine_size for details
+LARGE_MACHINE = { "gceMachineType" : "n1-standard-8"}
```

Comment:
It might be nicer to specify this in a .bzl file in the `//third_party/toolchains` directory in order to have all the platforms/toolchain stuff in one place.

Reply:
I'll try to do that in a follow up PR. There's some concerns about importability in google3.

Reply:
There's no symmetrical definition for google3. That abstraction isn't good enough.

Reply:
IIUC //third_party/toolchains is not imported into google3 either so putting it there should work? Before `exec_properties=LARGE_MACHINE` you used `exec_compatible_with=["//third_party/toolchains/machine_size:large"]` and presumably just had copybara remove it on import?